shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -220,57 +197,513 @@ - (CGRect)accessibilityFrame {
+  NSUInteger fromIndex = ((InactiveTextPosition*)fromPosition).index;
+  NSUInteger toIndex = ((InactiveTextPosition*)toPosition).index;
+  return [InactiveTextRange rangeWithNSRange:NSMakeRange(fromIndex, toIndex - fromIndex)];
+}
+
+- (UITextPosition*)positionFromPosition:(UITextPosition*)position offset:(NSInteger)offset {
+  return nil;
+}
+
+- (UITextPosition*)positionFromPosition:(UITextPosition*)position
+                            inDirection:(UITextLayoutDirection)direction
+                                 offset:(NSInteger)offset {
+  return nil;
+}
+
+- (NSComparisonResult)comparePosition:(UITextPosition*)position toPosition:(UITextPosition*)other {
+  return NSOrderedSame;
+}
+
+- (NSInteger)offsetFromPosition:(UITextPosition*)from toPosition:(UITextPosition*)toPosition {
+  return 0;
+}
+
+- (UITextPosition*)positionWithinRange:(UITextRange*)range
+                   farthestInDirection:(UITextLayoutDirection)direction {
+  return nil;
+}
+
+- (UITextRange*)characterRangeByExtendingPosition:(UITextPosition*)position
+                                      inDirection:(UITextLayoutDirection)direction {
+  return nil;
+}
+
+- (UITextWritingDirection)baseWritingDirectionForPosition:(UITextPosition*)position
+                                              inDirection:(UITextStorageDirection)direction {
+  return UITextWritingDirectionNatural;
+}
+
+- (void)setBaseWritingDirection:(UITextWritingDirection)writingDirection
+                       forRange:(UITextRange*)range {
+}
+
+- (CGRect)firstRectForRange:(UITextRange*)range {
+  return CGRectZero;
+}
+
+- (CGRect)caretRectForPosition:(UITextPosition*)position {
+  return CGRectZero;
+}
+
+- (UITextPosition*)closestPositionToPoint:(CGPoint)point {
+  return nil;
+}
+
+- (UITextPosition*)closestPositionToPoint:(CGPoint)point withinRange:(UITextRange*)range {
+  return nil;
+}
+
+- (NSArray*)selectionRectsForRange:(UITextRange*)range {
+  return @[];
+}
+
+- (UITextRange*)characterRangeAtPoint:(CGPoint)point {
+  return nil;
+}
+
+- (void)insertText:(NSString*)text {
+}
+
+- (void)deleteBackward {
+}
+
+@end
+
+@implementation TextInputSemanticsObject {
+  InactiveTextInput* _inactive_text_input;
+}
+
+- (instancetype)initWithBridge:(shell::AccessibilityBridge*)bridge uid:(int32_t)uid {
+  self = [super initWithBridge:bridge uid:uid];
+
+  if (self) {
+    _inactive_text_input = [[InactiveTextInput alloc] init];
+  }
+
+  return self;
+}
+
+- (void)dealloc {
+  [_inactive_text_input release];
+  [super dealloc];
+}
+
+#pragma mark - UIAccessibility overrides
+
+- (void)setSemanticsNode:(const blink::SemanticsNode*)node {
+  [super setSemanticsNode:node];
+  _inactive_text_input.text = @(node->value.data());
+  if ([self node].HasFlag(blink::SemanticsFlags::kIsFocused)) {
+    // The text input view must have a non-trivial size for the accessibility
+    // system to send text editing events.
+    [self bridge]->textInputView().frame = CGRectMake(0.0, 0.0, 1.0, 1.0);
```

Comment:
Just to confirm (and maybe to add to the comment): The textInputView itself is not focusable directly by the user, right? It remains hidden even though it has a size now?

Reply:
Done. See the `FlutterTextInputViewAccessibilityHider` class in https://github.com/flutter/engine/pull/4575/commits/07501fcb4d2a356d0160bffa486ee2439ce85a66.